docs/data_sources/data_catalog.rst

```diff
@@ -0,0 +1,126 @@
+.. _data-catalog:
```

Comment:
perhaps we can squish this page into top-level Data Sources page?

so it'd just be a list of links to the available data and these shorter descriptions of Work in Progress and Future Data?

Reply:
or this data-catalog page can be converted into a Future data/WIP page?? that might be cleaner

Reply:
ok I like the idea of Future/WIP for now!